detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/SandboxExecutor.kt

```diff
@@ -0,0 +1,59 @@
+package io.buildfoundation.bazel.rulesdetekt.wrapper
+
+import java.io.File
+import java.io.FileOutputStream
+import java.io.PrintStream
+import java.nio.charset.Charset
+import java.util.concurrent.atomic.AtomicReference
+
+interface SandboxExecutor {
```

Comment:
This naming might be confusing as Bazel has actual sandboxing feature :grimacing:

Reply:
Agree, but cannot come up with better ideas.